1224-user_administration/administrator_opens_user_administration.feature

```diff
@@ -0,0 +1,83 @@
+        And they see a section Registrator with a policy for who are able to publish
+        Examples:
+            | Role                       | Button            |
+            | Administrator              | Add Administrator |
+            | Curator                    | Add Curator       |
+            | Editor                     | Add Editor        |
+
+	@363
+    Scenario Outline: Administrator opens the Add Role Dialog
+        Given Administrator opens User Administration
+        When they click "<Button>" under "<Section>"
+        Then they see the Add Role Dialog 
+        And they see an Information box
+        And they see a input field to Search for employees
+        And they see a list of employees with an "Add" button
+        And they see the number of items viewed per page
+        And they can change the number of items viewed per page
+        And they see the number of items viewed of the total amount of items
+        And they see that previous page of items is disabled
+        And they can select next page of items
+        And they see a "Close" button
+        Examples:
+            | Section       | Button            |
+            | Administrator | Add Administrator |
+            | Curator       | Add Curator       |
+            | Editor        | Add Editor        |
+
+        @1362
+        Scenario: Administrator searches for User
+        Given Administrator opens the Add Role Dialog
+        When they enter text into the Search field
+        Then they see a list of employees matching the search with an "Add" button
+        And they see the number of items viewed per page
+        And they can change the number of items viewed per page
+        And they see the number of items viewed of the total amount of items
+        And they see that previous page of items is disabled
+        And they can select next page of items
+        And they see a "Close" button
+
+	@?
+    Scenario Outline: Administrator grants an employee a role
+        Given Administrator opens the Add Role Dialog
+        When the Administrator click the "Add" button under "<Section>"
+        Then the relevant employee is granted the role "<Role>"
+        And they see the Add Role Dialog 
+        And they see an Information box
+        And they see a input field to Search for employees
+        And they see a list of employees with an "Add" button
+        And they see the number of items viewed per page
+        And they can change the number of items viewed per page
+        And they see the number of items viewed of the total amount of items
+        And they see that previous page of items is disabled
+        And they can select next page of items
+        And they see a "Close" button
+        Examples:
+            | Section       | Role          |
+            | Administrator | Administrator |
+            | Curator       | Curator       |
+            | Editor        | Editor        |
```

Comment:
En del av dette er dekt i andre scenario. Tror vi bør endre litt for å forenkle testbarheten også, feks er `Then the relevant employee is granted the role "<Role>"` vanskelig å teste så vi vil heller teste hva som skal skje og er synlig for brukeren (@Eiriknil ?).

```suggestion
    Scenario: Administrator grants an Employee a role
        Given Administrator opens the Add Role Dialog
        When they click "Add" Button for an Employee
        Then they see a confirmation message
        And they see that the clicked "Add" button is disabled
```